sherpa/utils/tests/test_utils_unit.py

```diff
@@ -0,0 +1,203 @@
+#
+#  copyright (c) 2016  smithsonian astrophysical observatory
```

Comment:
Another "picky" comment: what has happened to all the capital letters in this section?